src/libs/blueprint/conduit_blueprint_mesh_examples.cpp

```diff
@@ -3421,6 +3421,165 @@ adjset_uniform(Node &res)
+        values_x[i * 20 + 4] = 1 + i * 2;  values_y[i * 20 + 4] = -1;   values_z[i * 20 + 4] = 1 + i * 2;
+        values_x[i * 20 + 5] = 1 + i * 2;  values_y[i * 20 + 5] = -1;   values_z[i * 20 + 5] = -1 + i * 2;
+        values_x[i * 20 + 6] = -1 + i * 2; values_y[i * 20 + 6] = -1;   values_z[i * 20 + 6] = -1 + i * 2;
+        values_x[i * 20 + 7] = -1 + i * 2; values_y[i * 20 + 7] = -1;   values_z[i * 20 + 7] = 1 + i * 2;
+
+        // points for half the triangular prisms
+        values_x[i * 20 + 8] = 1 + i * 2;       values_y[i * 20 + 8] = 1;   values_z[i * 20 + 8] = -1 + i * 2 + 2;
+        values_x[i * 20 + 9] = -1 + i * 2;      values_y[i * 20 + 9] = 1;   values_z[i * 20 + 9] = -1 + i * 2 + 2;
+        values_x[i * 20 + 10] = -1 + i * 2 + 2; values_y[i * 20 + 10] = 1;  values_z[i * 20 + 10] = 1 + i * 2 + 2;
+        values_x[i * 20 + 11] = 1 + i * 2;      values_y[i * 20 + 11] = -1; values_z[i * 20 + 11] = -1 + i * 2 + 2;
+        values_x[i * 20 + 12] = -1 + i * 2;     values_y[i * 20 + 12] = -1; values_z[i * 20 + 12] = -1 + i * 2 + 2;
+        values_x[i * 20 + 13] = -1 + i * 2 + 2; values_y[i * 20 + 13] = -1; values_z[i * 20 + 13] = 1 + i * 2 + 2;
+
+        // points for the other half
+        values_x[i * 20 + 14] = 1 + i * 2;      values_y[i * 20 + 14] = 1;  values_z[i * 20 + 14] = -1 + i * 2 + 2;
+        values_x[i * 20 + 15] = -1 + i * 2 + 2; values_y[i * 20 + 15] = 1;  values_z[i * 20 + 15] = -1 + i * 2;
+        values_x[i * 20 + 16] = -1 + i * 2 + 4; values_y[i * 20 + 16] = 1;  values_z[i * 20 + 16] = 1 + i * 2;
+        values_x[i * 20 + 17] = 1 + i * 2;      values_y[i * 20 + 17] = -1; values_z[i * 20 + 17] = -1 + i * 2 + 2;
+        values_x[i * 20 + 18] = -1 + i * 2 + 2; values_y[i * 20 + 18] = -1; values_z[i * 20 + 18] = -1 + i * 2;
+        values_x[i * 20 + 19] = -1 + i * 2 + 4; values_y[i * 20 + 19] = -1; values_z[i * 20 + 19] = 1 + i * 2;
+    }
+
+    chain_topo["type"] = "unstructured";
+    chain_topo["coordset"] = "coords";
+    chain_topo["elements/shape"] = "polyhedral";
+
+    // 6 cube faces + 5 prism faces * 2 prisms makes 16
+    chain_topo["elements/connectivity"].set(conduit::DataType::int64(howmany * 16));
+    int64* connec = chain_topo["elements/connectivity"].value();
+    for (int i = 0; i < howmany * 16; i ++)
+    {
+        // our faces are specified in order and no faces are reused
+        connec[i] = i;
+    }
+
+    // this is 3 because every time howmany is increased by 1, 3 more polyhedra are added,
+    // the cube and two prisms
+    chain_topo["elements/sizes"].set(conduit::DataType::int64(howmany * 3));
+    int64* sizes = chain_topo["elements/sizes"].value();
+    for (int i = 0; i < howmany * 3; i ++)
+    {
+        // this ensures that sizes will be of the form {6,5,5, 6,5,5, 6,5,5, ..., 6,5,5}
+        sizes[i] = ((i % 3) > 0) ? 5 : 6;
+    }
+
+    chain_topo["elements/offsets"].set(conduit::DataType::int64(howmany * 3));
```

Comment:
To simplify this, you can use the existing helper function [`blueprint::mesh::topology::unstructured::generate_offsets`](https://github.com/LLNL/conduit/blob/develop/src/libs/blueprint/conduit_blueprint_mesh.hpp#L362).